### DISCUSSION TITLE
Desynthesis increase messages are hidden

### DISCUSSION THREAD

**Discussion by StellarJay77** [NadyaNayme/TidyChat#28] [Q&A] [answered]
Desynthesis increase messages are hidden and no selection to enable that I can find. Is there one?

**Comment by NadyaNayme** [comment]
There isn't a filter for it - but I'll get a show-filter added if you like.

Desynth level is basically meaningless now since it only affects chance of receiving an HQ mat in return. Good for those going for true completion I suppose. So I didn't think to whitelist it.

**Reply by StellarJay77** [reply]
Yes, I would like to see the option to show it at least. I do desynth items when they potentially give an increase to it  so I am better able to get rare mats from it.

**Reply by NadyaNayme** [reply]
You misunderstand how the desynth bonus works - the bonus isn't for _rare mats_. It's for _HQ mats_ which makes it pretty worthless with modern crafting which requires very few, if any, HQ mats. The game isn't very clear on that rare=hq so people are misled that rare=stuff like the swimsuit halters.

**Reply by StellarJay77** [reply]
No I get it however there's a few mats that you have a higher chance of getting to craft some things (eg. perfect pestle and perfect firewood for ARR relics) but that I also want to max out my desynthesis values and not seeing them with tidy chat on makes it difficult to establish if it it increasing.

**Reply by NadyaNayme** [reply]
To the best of my knowledge it has no impact on the rate of obtaining those items.

I'll get the filter added but I still stand on it being essentially worthless with how desynth currently works.

**Comment by NadyaNayme** [comment, marked as answer]
The filter is currently waiting for PR: https://github.com/goatcorp/DalamudPlugins/pull/2083
